doc/source/setup/security.rst

```diff
@@ -39,6 +44,14 @@ Grafana
 Enable authentication
 =====================
 
+CrateDB
+-------
+
+.. todo::
+
+    Outline how to enable authentication with CrateDB, and how to amend the default
+    credentials.
```

Comment:
I would like to demonstrate here how to create a new user account, and how to use it to connect to CrateDB. Are there any tutorials around about this, for educating users?

### Documentation
- https://crate.io/docs/crate/reference/en/5.3/admin/auth/methods.html
- https://crate.io/docs/crate/reference/en/5.3/admin/auth/hba.html
- https://crate.io/docs/crate/reference/en/5.3/admin/user-management.html
- https://crate.io/docs/crate/reference/en/5.3/sql/statements/create-user.html

### Resources
- https://community.crate.io/t/password-for-crate-user/194